### DISCUSSION TITLE
WebUI Errors

### DISCUSSION THREAD

**Discussion by ccw2777** [intelowlproject/IntelOwl#1721] [Q&A]
Hello! Any advice would be appreciated as I have not found this issue discussed in other threads. After creating a super user, every time I use the account to log into the webUI I receive the errors seen in the below screenshots. The fresh install of the OS Ubuntu appeared to go perfectly without issues so I am at a loss as to whats the root cause. I followed the install guide exactly. 
Only thing of note is that the command seen in the Terminal screenshot is where it normally stays for long periods of time,  I dont know if its supposed to complete fully in the terminal. 
![login error](https://github.com/intelowlproject/IntelOwl/assets/10868403/1736cbe0-12ec-40a1-8752-1dadf1d67dcd)
![WebUI Error](https://github.com/intelowlproject/IntelOwl/assets/10868403/06c0a04c-7d23-45dc-8422-b4cb6bc79c8f)
![WebUI Error2](https://github.com/intelowlproject/IntelOwl/assets/10868403/553a0bd3-4ff4-4058-954d-6ded0eedc7fb)
![terminal screenshot](https://github.com/intelowlproject/IntelOwl/assets/10868403/0a5d810b-be13-4e6f-9671-64e183b8b9c8)

**Comment by mlodic** [comment]
moving the issue from https://github.com/intelowlproject/IntelOwl/issues/1710 to here, in particular when 2 different maintainers are trying to help you and you did not answer, it is not convenient at all and you are making us waste our time. 

This issue is not reproducible and from our side everything works as expected. Most probably is something related your network or OS configuration which we cannot know. We need application logs to understand where those 500 errors are generated. You can get them by following the guide here: https://intelowl.readthedocs.io/en/latest/Contribute.html#debug-application-problems

**Reply by ccw2777** [reply]
I understand.  I mentioned this question in #1721, is there a way to get IntelOwl to skip Tor for the time being so that it no longer hangs when trying to access check.torproject.org?  I attempted just disabling Tor analyzer in the admin console and booting IntelOwl down and then back up but now with Tor analyzer" disabled it just hangs as shown below. Not sure of the best way to get Tor as a whole. 
![Tor issue](https://github.com/intelowlproject/IntelOwl/assets/10868403/0dfdadc2-4b5e-4302-93f4-a99a8a1bc494)

**Comment by mlodic** [comment]
Closing this because duplicated from https://github.com/intelowlproject/IntelOwl/issues/1710
